apps/api/src/app/auth/auth.guard.ts

```diff
@@ -27,15 +27,19 @@ export class AuthGuard implements CanActivate {
     const request = context.switchToHttp().getRequest();
     const token = this.extractTokenFromHeader(request);
     if (!token) {
-      throw new UnauthorizedException();
+      throw new UnauthorizedException(
+        `Failled to exctactToken or token invalide`
```

Comment:
Fréro ?

Reply:
Oui mon ref. Le message d'erreur te dérange ?

Reply:
Faut se relire :) Dans le pire des cas demande à GPT mais quand même 🤡

Reply:
Failed to extractToken or token invalid$

Reply:
Haha t'es un gros connard. T'abuses, tu me mets des balleyellets juste pour un e 🤣😂. T'es pas prêt toi !